src/npm.rs

```diff
@@ -560,7 +554,7 @@ impl NpmPackageNv {
   }
 
   pub fn scope(&self) -> Option<&str> {
-    if self.name.starts_with('@') && self.name.contains('/') {
```

Comment:
No need to check this. I verified by searching all the existing npm package names.